src/i18n/locales/en.json

```diff
@@ -46,6 +46,8 @@
   "requiredField": "This field is required.",
   "minLength": "This field must be at least {{field}} characters.",
   "maxLength": "This field must be at most {{field}} characters.",
+  "notSpecialCharacters": "A text containing special characters cannot be sent.",
+  "notOnlyNumbers": "This field cannot consist only of numbers.",
```

Comment:
Text containing a special character cannot be sent.
bence daha iyi?

Reply:
Düzelttim